test/functional/change_stream_tests.js

```diff
@@ -36,6 +36,41 @@ describe('Change Streams', function() {
       .then(() => client.close(), () => client.close());
   });
 
+  it('Should close the listeners after the cursor is closed', {
+    metadata: { requires: { topology: 'replicaset', mongodb: '>=3.5.10' } },
+
+    // The actual test we wish to run
+    test: function(done) {
+      let closed = false;
+      const close = _err => {
+        if (closed) {
+          return;
+        }
+        closed = true;
+        return client.close(() => done(_err));
+      };
+      const configuration = this.configuration;
+      const client = configuration.newClient();
+
+      client.connect(function(err, client) {
```

Comment:
Can we use an arrow function here so `function(err, client) {` becomes `(err, client) => {`?